frontend/src/pages/projects/components/NameDescriptionField.tsx

```diff
@@ -80,7 +80,7 @@ const NameDescriptionField: React.FC<NameDescriptionFieldProps> = ({
               isDisabled={disableK8sName}
               id={`resource-${nameFieldId}`}
               name={`resource-${nameFieldId}`}
-              aria-labelledby={`resource-${nameFieldId}`}
+              aria-label={`resource-${nameFieldId}`}
```

Comment:
This is a valid label? Seems a bit technical no?

Reply:
Nice catch - it turns out aria-label or aria-labelledby isn't necessary here because the `fieldId` of  PF `FormGroup` needs to match the id of PF `TextInput`. Once this is the case, the screenreader should read out the label of the field without needing any hard-coded ARIA attributes.